plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/gramine/GenerateGramineBundle.kt

```diff
@@ -63,15 +66,33 @@ open class GenerateGramineBundle @Inject constructor(
 
     override fun action() {
         enclaveJar.copyToOutputDir(GRAMINE_ENCLAVE_JAR)
+
+        val manifestTemplatePath = temporaryDir.resolve(MANIFEST_TEMPLATE).toPath()
+        javaClass.copyResource(MANIFEST_TEMPLATE, manifestTemplatePath)
+
         if (pythonFile.isPresent) {
-            pythonFile.copyToOutputDir(PYTHON_FILE)
+            generateManifestForPythonEnclaves(manifestTemplatePath.absolutePathString())
+        } else {
+            generateManifestForJavaEnclaves(manifestTemplatePath.absolutePathString())
```

Comment:
Please use `Path` as much as possible, only converting to strings where necessary.

Reply:
When this is used I actually need this as a string. Therefore I preferred to pass around the actual think I needed. Not a big deal anyway, reverted to be a path and then converting at the end